extensions/stackdriver/config/v1alpha1/stackdriver_plugin_config.proto

```diff
@@ -68,4 +68,7 @@ message PluginConfig {
   // metrics for HTTP traffic. Defaults to false (request and response size
   // metrics are enabled).
   bool disable_http_size_metrics = 8;
+
+  // Optional. Allows enabling of reporting of the metrics for TCP traffic.
+  bool enable_tcp_metrics = 9;
```

Comment:
I am wondering do we need this option. We can always record and send the TCP metrics when this this filter is added to a network filter chain. The other thing that this option controls is whether to set up timer. I think we can also just start timer regardless of this option or other enablement. i.e. replace
```
if (enableServerAccessLog() || enableEdgeReporting() || enableTCPMetrics()) {
  proxy_set_tick_period_milliseconds(getLoggingExportIntervalMilliseconds());
} else {
  ...
}
```
with
```
proxy_set_tick_period_milliseconds(getLoggingExportIntervalMilliseconds());
```

Reply:
What about the scenario if we want to get TCP connection metrics for HTTP?

Reply:
That will also require to inject this filter into listener filter chains.

Reply:
Agreed to that point.. but I thought we wanted to disable metrics till we have a stackdriver schema done for TCP.. and we can control addition of this filter via a bool in Istio but then we will not be able to get TCP access log too, if we control just addition of this filter in listener filter chains.. it would be good to have an option to enable metrics and logs separately..

Reply:
As discussed offline.. keeping it for now and will remove it once we have schema ready in stackdriver.

Reply:
sorry to reopen this, but I am still hesitate to add this for temporary state. Can we just comment out invocation of recordTCP() (or any function that triggers connection stats recording) when cherry-pick to 1.6 and make a new build when TCP metric is ready?

Reply:
Done..